Makefile

```diff
@@ -95,7 +96,9 @@ bottles-macos: | $(BOTTLE_OPENSSL) $(BOTTLE_PCRE)
 	rm -rf bottles/Downloads
 
 ifeq ($(detected_OS), Darwin)
- NIM_PARAMS := $(NIM_PARAMS) -L:"-framework Foundation -framework Security -framework IOKit -framework CoreServices"
+NIM_PARAMS += -L:"-framework Foundation -framework Security -framework IOKit -framework CoreServices"
+# Fix for failures due to 'can't allocate code signature data for'
+NIM_PARAMS += --passL:"-headerpad_max_install_names"
```

Comment:
Why not keep these lines indented between `ifeq` / `endif`?

Reply:
Sure, why not.